components/eventing-controller/testing/v2/test_helpers.go

```diff
@@ -242,6 +244,17 @@ func NewBEBSubscription(name, contentMode string, webhookURL string, events type
 	}
 }
 
+func NewSampleEventMeshSubscription() *types.Subscription {
+	eventType := []types.Event{
+		{
+			Source: EventSource,
```

Comment:
(nit) maybe not clean source could also indirectly test the source cleaning?

Reply:
its already covered in multiple place.